molecule/commands/converge.py

```diff
@@ -82,14 +81,8 @@ def execute(self,
         # install role dependencies only during `molecule converge`
         if not idempotent and 'requirements_file' in self.molecule.config.config[
                 'ansible']:
-            utilities.print_info('Installing role dependencies ...')
-            galaxy_install = ansible_galaxy_install.AnsibleGalaxyInstall(
-                self.molecule.config.config['ansible']['requirements_file'])
-            galaxy_install.add_env_arg(
-                'ANSIBLE_CONFIG',
-                self.molecule.config.config['ansible']['config_file'])
-            galaxy_install.bake()
-            output = galaxy_install.execute()
+            self.molecule.download_dependencies(self.molecule.config.config[
```

Comment:
This feels like it belongs in the ansible_galaxy install class vs core.

We should simply call `ansible_galaxy_install.install()` or `ansible_galaxy_install.download()`.  Core IMO is too large and largely untested (#337).